index.js

```diff
@@ -1,34 +1,74 @@
 'use strict';
 const arrify = require('arrify');
 const taskkill = require('taskkill');
+const tasklist = require('tasklist');
 const execa = require('execa');
 const AggregateError = require('aggregate-error');
 
-function win(input, opts) {
+function winKillProcess(input, opts) {
 	return taskkill(input, {
 		force: opts.force,
 		// Don't kill ourselves
 		filter: `PID ne ${process.pid}`
 	});
 }
 
-function def(input, opts) {
-	const cmd = typeof input === 'string' ? 'killall' : 'kill';
+function winKill(input, opts) {
+	const killByName = typeof input === 'string';
+	if (killByName && opts.ignoreCase) {
+		return tasklist()
+			.then(tasks => tasks.filter(task => task.imageName.toLowerCase() === input.toLowerCase()))
+			.then(names => Promise.all(names.map(name => winKillProcess(name, opts))));
```

Comment:
`taskkill` also supports an array of strings as input. I choose not to use it to keep the function more consistent with the rest.